docs: update README.md with install and feature sections (#61)
slackapi/slack-mcp-plugin: README.md

# Slack MCP and Skills Plugin

> A plugin for [Claude Code][claude-code] and [Cursor][cursor] that brings Slack into your AI assistant - through the [Slack MCP server][slack-mcp-docs] for direct workspace actions and a curated set of Slack skills for both users and developers.

Reviewer: This is just my suggestion but I think it could be shorter 🤔 

```suggestion
A plugin that brings Slack into your AI tools via the [Slack MCP server][slack-mcp-docs] and provides a set of Slack skills for users and developers.
```

Author: I hear you on a shorter description and I like your use of "tools" and "provides". However, I'm going to push back a little on dropping "Claude Code" and "Cursor." The term "plugin" is very generic and until these tools become more standard, I think we should continue to be specific with "Claude Code plugin" and "Cursor plugin" for discoverability. 

```suggestion
A [Claude Code][claude-code] and [Cursor][cursor] plugin that brings Slack into your AI tools with a [Slack MCP Server][slack-mcp-docs] and set of Slack skills for both users and developers.
```

Author: The above takes the best of both, I believe. It shortens the intro, removes buzz words, while keeping references to Claude Code and Cursor. Over time, we can move the Claude Code and Cursor into a supported section and just call this a skills plugin or AI Coding Tool plugin - whatever emerges as the common term.

[![CI Build](https://github.com/slackapi/slack-mcp-plugin/actions/workflows/ci-build.yml/badge.svg)](https://github.com/slackapi/slack-mcp-plugin/actions/workflows/ci-build.yml)
[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)

## Installation

### Claude Code

The plugin is published on the [official Claude marketplace](https://claude.com/plugins/slack). Install it from inside Claude Code:

```
/plugin install slack@claude-plugins-official
```

The Slack MCP server is configured automatically. You'll be prompted to authenticate to your Slack workspace via OAuth on first use.

### Cursor

Click the button below to add the Slack MCP server to Cursor:

[![Install MCP Server](https://cursor.com/deeplink/mcp-install-dark.svg)](https://cursor.com/en-US/install-mcp?name=slack&config=eyJ1cmwiOiJodHRwczovL21jcC5zbGFjay5jb20vbWNwIiwiYXV0aCI6eyJDTElFTlRfSUQiOiIzNjYwNzUzMTkyNjI2Ljg5MDM0NjkyMjg5ODIifX0%3D)

After install, Cursor surfaces a connect button - use it to authenticate to your Slack workspace.

## Features

### MCP Server

> The plugin connects your AI assistant to Slack's hosted [MCP server][slack-mcp-docs]:

Reviewer: What do you think about using the `tool` terminology? 🤔 

```suggestion
The plugin connects your AI tool to Slack's hosted [MCP server][slack-mcp-docs]:
```

Author: Good suggestion, I think "tool" is more future-proof and flexible.

- [`slack:slack-search`](skills/slack-search/SKILL.md) - finding messages, files, channels, and people
- [`slack:slack-api`](skills/slack-api/SKILL.md) - discovering and calling Slack Web API methods
- [`slack:slack-cli`](skills/slack-cli/SKILL.md) - using the [Slack CLI][slack-cli] to create, run, and manage apps
- [`slack:create-slack-app`](skills/create-slack-app/SKILL.md) - building a Slack app or agent with the CLI and [Bolt][bolt]
- [`slack:block-kit`](skills/block-kit/SKILL.md) - building and validating [Block Kit][block-kit] layouts

### Commands

Five slash commands for common Slack workflows:

- `/slack:summarize-channel <channel-name>` - Summarize recent activity in a Slack channel
- `/slack:find-discussions <topic>` - Find discussions about a specific topic across Slack channels
- `/slack:draft-announcement <topic>` - Draft a well-formatted Slack announcement and save it as a draft
- `/slack:standup` - Generate a standup update based on your recent Slack activity
- `/slack:channel-digest <channel1, channel2, ...>` - Get a digest of recent activity across multiple Slack channels

## Usage examples

> Once installed, talk to your assistant in natural language:

Reviewer: ```suggestion
Once installed, talk to your tool in natural language:
```

Author: Works for me! Thanks

- "Search for messages about the product launch from the last week"
- "Send a message to #general saying the deployment is complete"
- "Summarize the last day of activity in #engineering"
- "Draft an announcement about the new pricing page"
- "Create a new Slack app using Bolt for Python"
- "Build a Block Kit feedback modal with a rating select and a comments field"
- "Validate the Block Kit JSON in ./modal.json"

## Documentation

- [Slack MCP server][slack-mcp-docs]
- [Slack developer docs](https://docs.slack.dev/)
- [Block Kit Builder][block-kit]

## Limitations

- **Workspace admin approval.** Your Slack workspace admin must approve MCP integration before you can authenticate.

## Contributing

> We welcome contributions from everyone! Please check out our [contributor's guide](.github/contributing.md) for guidelines on opening issues and pull requests.

Reviewer: ➕

> Working on the plugin itself? See the [maintainer's guide](.github/maintainers_guide.md) for local development setup.

Reviewer: Praise 🙏
